src/main/kotlin/lotto/model/LottoNumber.kt

```diff
@@ -0,0 +1,19 @@
+package lotto.model
+
+/**
+ * 로또 숫자 하나의 객체
+ * */
+@JvmInline
+value class LottoNumber(val number: Int) {
+    init {
+        require(number in MIN_NUMBER..MAX_NUMBER) { EXCEPTION_NUMBER_FORMAT }
+    }
+
+    companion object {
+        const val MIN_NUMBER = 1
+        const val MAX_NUMBER = 45
```

Comment:
```suggestion
        private const val MIN_NUMBER = 1
        private const val MAX_NUMBER = 45
```
내부에서 사용하는 변수는 private 접근자를 반드시 넣어주세요